2021/27xxx/CVE-2021-27856.json

```diff
@@ -1,18 +1,125 @@
 {
```

Comment:
'problemtype' is a required property

Reply:
Fixed in https://github.com/CVEProject/cvelist/pull/3715/commits/7f6e785d430425b527782b8ec350c283baeca1f4